huaweicloud/services/elb/resource_huaweicloud_elb_l7rule.go

```diff
@@ -42,42 +41,49 @@ func ResourceL7RuleV3() *schema.Resource {
 				Computed: true,
 				ForceNew: true,
 			},
-
+			"l7policy_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
 			"type": {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
-				ValidateFunc: validation.StringInSlice([]string{
-					"HOST_NAME", "PATH",
-				}, true),
 			},
-
 			"compare_type": {
 				Type:     schema.TypeString,
 				Required: true,
-				ValidateFunc: validation.StringInSlice([]string{
-					"STARTS_WITH", "EQUAL_TO", "REGEX",
-				}, true),
 			},
-
-			"l7policy_id": {
+			"value": {
 				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
+				Optional: true,
+				Computed: true,
+			},
+			"conditions": {
```

Comment:
is it ok when both `valve` and`conditions` are empty?

Reply:
fixed，add atLeastOneOf